CONTRIBUTING.md

```diff
@@ -81,6 +83,13 @@ https://github.com/pytorch/pytorch#from-source
 
 To develop PyTorch on your machine, here are some tips:
 
+### Prerequisites
+* CMake. We recommend installing it with [Homebrew](https://brew.sh/)
+with `brew install cmake` if you are developing on MacOS or Linux system.
```

Comment:
```suggestion
with `brew install cmake` if you are developing on MacOS.
```

I'd probably reword this to suggest installing it through their package manager since using brew on a linux box seems like an anti-pattern when compared to macOS

Reply:
Makes sense. Updated!